draft-ietf-anima-brski-prm.md

```diff
@@ -1694,117 +1751,153 @@ This results in a signed CA certificate(s) object (JSON-in-JWS), the CA certific
-* enrollment-response - LDevID (Pledge) certificate (from CA via registrar)
+## Supply Voucher to Pledge {#voucher}
 
-To deliver these response objects, the Registrar-Agent will re-connect to the pledge.
-To contact the pledge, it may either discover the pledge as described in {{discovery_uc2_ppa}} or use stored information from the first contact with the pledge.
+Once the Registrar-Agent has acquired the following three bootstrapping artifacts, it can supply them to the pledge starting with the Voucher':
 
-Preconditions in addition to {{pvr}}:
+* Voucher': voucher countersigned by the registrar (from MASA via Registrar)
+* Enroll-Resp: pledge LDevID certificate (from Key Infrastructure via registrar)
+* caCerts: domain trust anchors (from Key Infrastructure via Registrar)
 
-* Registrar-Agent: obtained voucher and LDevID certificate and optionally IDevID CA certificates.
-  The IDevID CA certificate is necessary, when the connection between the Registrar-Agent and the pledge is established using TLS to enable the Registrar-Agent to validate the pledges' IDevID certificate during the TLS handshake as described in {{tpvr}}.
+Reconnecting to the pledge might require to re-discover the pledge as described in {{discovery_uc2_ppa}}.
+The Registrar-Agent MAY stored information from the first connection with the pledge to optimize.
 
-The Registrar-Agent MAY optionally use TLS to protect the communication as outlined in {{tpvr}}.
+Optionally, TLS MAY be used to provide privacy for this exchange between the Registrar-Agent and the pledge (see {{pledgehttps}}).
 
-The Registrar-Agent provides the information via distinct pledge endpoints as following.
-{{exchangesfig_uc2_6}} shows the provisioning of the voucher to the pledge. 
-The following subsections describe the corresponding artifacts. 
+
+{{exchangesfig_uc2_6}} shows the provisioning of the voucher to the pledge and the following subsections describe the corresponding artifacts.
 
 ~~~~ aasvg
 +--------+    +------------+    +-----------+    +--------+    +------+
-| Pledge |    | Registrar- |    |  Domain   |    | Domain |    | MASA |
-|        |    |   Agent    |    | Registrar |    |   CA   |    |      |
+| Pledge |    | Registrar- |    |  Domain   |    |  Key   |    | MASA |
+|        |    |   Agent    |    | Registrar |    | Infra. |    |      |
 +--------+    +------------+    +-----------+    +--------+    +------+
  |                  |                 |                 |   Internet |
  ~                  ~                 ~                 ~            ~
 (6) Supply Voucher to Pledge
  ~                  ~                 ~                 ~            ~
  |                  |                 |                 |            |
  |<----opt. TLS---->|                 |                 |            |
- |<-----Voucher-----|                 |                 |            |
+ |<-----Voucher''---|                 |                 |            |
```

Comment:
Note that it takes two apostrophes to render one via Kramdown.

Reply:
okay, thanks for the info